src/components/ListOfTodos.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+
+import { TodoComponent } from './TodoComponent';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filterCallback: (todo: Todo) => boolean,
+};
+
+export const ListOfTodos: FC<Props> = ({
+  todos,
+  filterCallback,
+}) => (
+  <section
+    className="todoapp__main"
+  >
+    {todos.filter(filterCallback)
+      .map(({ id, title, completed }: Todo) => (
+        <TodoComponent
+          id={id}
+          completed={completed}
+          title={title}
+        />
```

Comment:
don't forget to add `key` by rendering a list of components
```suggestion
        <TodoComponent
          key={id}
          id={id}
          completed={completed}
          title={title}
        />
```